tools/commentMonitor/main.go

```diff
@@ -31,26 +31,64 @@ import (
 	"gopkg.in/alecthomas/kingpin.v2"
 )
 
+type githubClient struct {
+	clt      *github.Client
+	owner    string
+	repo     string
+	prnumber int
```

Comment:
it is quite clear that it will be a number.

```suggestion
	pr int
```